src/Workspaces/CSharp/Portable/CodeStyle/CSharpCodeStyleOptions.cs

```diff
@@ -12,79 +12,79 @@ internal static class CSharpCodeStyleOptions
         public static readonly Option<CodeStyleOption<bool>> UseImplicitTypeForIntrinsicTypes = new Option<CodeStyleOption<bool>>(
             nameof(CodeStyleOptions), nameof(UseImplicitTypeForIntrinsicTypes), defaultValue: CodeStyleOption<bool>.Default,
             storageLocations: new OptionStorageLocation[] {
-                new EditorConfigStorageLocation("csharp_style_var_for_built_in_types", ParseEditorConfigCodeStyleOption),
+                new EditorConfigStorageLocation("csharp_style_var_for_built_in_types"),
```

Comment:
Move `ParseEditorConfigCodeStyleOption` closer to `EditorConfigStorageLocation`.

Also consider caching the delegate for the parse function used in the 1-arg version of the `EditorConfigStorageLocation` constructor